### DISCUSSION TITLE
The `getredash/redash` project has basically stopped, and we can now continue the new journey of Redash in a new place.  https://github.com/RedashCommunity/redash

### DISCUSSION THREAD

**Discussion by Jengro777** [getredash/redash#6121] [Q&A]
It is already difficult to merge the code here. You have another option available:
https://github.com/RedashCommunity/redash

**Comment by arikfr** [comment]
There was some discussion about this here: https://github.com/getredash/redash/discussions/5962#discussioncomment-6263667.
